examples/slo/main.tf

```diff
@@ -0,0 +1,9 @@
+# NOTE! to run the example move config folder to modules/slos/ folder!
+
+module slos {
+  source = "../../modules/slos"
+
+  display_name          = "my-service"
+  monitoring_project_id = "monitoring-project-id"
+  project_env           = "prod"
```

Comment:
We don't use `project_env` in the module. We can strict the environment in the CLI. Module needs to be more generic